### DISCUSSION TITLE
Why is layoutDirection hardcoded to leftToRight on iPhones?

### DISCUSSION THREAD

**Discussion by Jelizondo** [pointfreeco/swift-snapshot-testing#802] [Q&A]
Hello!

When generating snapshots I noticed that layout direction is always left to tight on iPhones but not on iPads! I'm wondering why the decision to do this? I'll be happy to create a PR if all agree that this can be removed! Thanks!

Example found on View.swift file:
```swift
        public static func iPhoneX(_ orientation: ViewImageConfig.Orientation)
          -> UITraitCollection
        {
          let base: [UITraitCollection] = [
            .init(forceTouchCapability: .available),
            .init(layoutDirection: .leftToRight),
            .init(preferredContentSizeCategory: .medium),
            .init(userInterfaceIdiom: .phone),
          ]
          switch orientation {
          case .landscape:
            return .init(
              traitsFrom: base + [
                .init(horizontalSizeClass: .compact),
                .init(verticalSizeClass: .compact),
              ]
            )
          case .portrait:
            return .init(
              traitsFrom: base + [
                .init(horizontalSizeClass: .compact),
                .init(verticalSizeClass: .regular),
              ]
            )
          }
        }
```

**Comment by nikitaame** [comment]
For context, this is giving us inaccurate results in Snapshotting RTL languages. We were going to make our own devices in our codebase but wanted to first understand the reasoning for this decision. The same code *was* once present in iPad's but has been commented out 🤔

**Comment by Jelizondo** [comment]
Hello again! I would appreciate an answer here! Is this something we could improve? I'll be happy to create a PR if all agree that this can be removed!

**Reply by mbrandonw** [reply]
Hi @Jelizondo, yeah I think you are right. Feel free to PR to remove. Thanks!
